packages/notifi-react-card/lib/hooks/useFetchedCardState.ts

```diff
@@ -22,14 +22,20 @@ export type VerifyOnboardingViewState = Readonly<{
   state: 'verifyonboarding';
 }>;
 
+export type ErrorViewState = Readonly<{
+  state: 'error';
+  reason: unknown;
```

Comment:
Ah, I guess this is reusing the same pattern as the "fetch" state error card

I think the error card just renders the `reason` as a string -- it would be nice if we can update the logic in ErrorCard a little bit so we can format the errors a little bit more nicely.

Reply:
ill do it now